test/ejabberd_tests/tests/ejabberdctl_SUITE.erl

```diff
@@ -595,10 +610,188 @@ stats_host(Config) ->
                 {"0\n", 0} = ejabberdctl("stats_host", ["onlineusers", SecDomain], Config)
         end).
 
+
+
+%%-----------------------------------------------------------------
+%% Improve coverage
+%%-----------------------------------------------------------------
+
+
+
+simple_register(Config) ->
+    %% given
+    {_, Domain, _} = get_user_data(kate, Config),
```

Comment:
`escalus_ct:get_config(ejabberd_domain)` seams to be more convenient to get the server's domain. Unless `kate` has different domain then the default one and this is what we want.